backend/__tests__/integration_tests/test_application.py

```diff
@@ -25,6 +26,7 @@ def setUp(self):
 
     @mock_authentication_context
     def test_new_application_without_partner(self):
+        createBasicSeason(db.session)
```

Comment:
Is this used for anything? The test doesn't expect to receive a connected season, but that might be intended behaviour.

Reply:
The applicationService fetches getCurrentOrNext so a applicationSeason is required.